metricbeat/docker-compose.yml

```diff
@@ -91,3 +105,8 @@ services:
 
   zookeeper:
     image: jplock/zookeeper:3.4.8
+
+volumes:
```

Comment:
If these volumes are ceph specific, we should have better names. If possible, I would prefer not to have "global" volumes.